### PR TITLE
Fixing fcl paths in POMS

### PR DESCRIPTION
**Pull request by soleti**
Few changes to the POMS configurations:
- The paths for the resampler fcl files reflect the recent changes
- We store FCL files and logs on persistent (not on scratch)
- We run only onsite

I also made a small change to the PBI script, which now produces also the JSON file.

### CODE REVIEW

File: CampaignConfig/mdc2020_corsika.cfg [hunk 1]
```diff
@@ -132,7 +131,7 @@ job_setup.postscript_1 = samweb file-lineage parents `basename ${fname}` > paren
 job_setup.postscript_2 = CORSIKA_EXE=`which corsika77400Linux_QGSJET_fluka`
 job_setup.postscript_3 = DATDIR=`dirname $CORSIKA_EXE`
 job_setup.postscript_4 = IFS='.' read -r -a FIELDS <<< "`cat filename.txt`"
-job_setup.postscript_5 = sed -e "s:_DATDIR_:$DATDIR/:" -e "s:_DIRECT_:`pwd`/:" -e "s:_SEED1_:$RANDOM:" -e "s:_SEED2_:$RANDOM:" -e "s:_NSHOW_:3000000:" -e "s:RUNNR   1:RUNNR   ${FIELDS[4]}:" /cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/current/link/Production/JobConfig/cosmic/CORSIKAInput.txt > corsika_conf.txt
+job_setup.postscript_5 = sed -e "s:_DATDIR_:$DATDIR/:" -e "s:_DIRECT_:`pwd`/:" -e "s:_SEED1_:$RANDOM:" -e "s:_SEED2_:$RANDOM:" -e "s:_NSHOW_:3000000:" -e "s:RUNNR   1:RUNNR   ${FIELDS[4]}:" /cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/%(release)s/link/Production/JobConfig/cosmic/CORSIKAInput.txt > corsika_conf.txt
```
Comment (brownd1978) [inline CampaignConfig/mdc2020_corsika.cfg:134]: is the 's' after %(release) intentional?

**Review by brownd1978 — COMMENTED** [review]
One minor question, otherwise this looks good.

File: CampaignConfig/mdc2020_corsika.cfg (hunk 1 above)
Comment (soleti) [reply CampaignConfig/mdc2020_corsika.cfg:134]: Yes, it's the format of the ini variables

Comment (brownd1978) [reply CampaignConfig/mdc2020_corsika.cfg:134]: It doesn't look like the others: its inside the directory path, not an argument.

Comment (soleti) [reply CampaignConfig/mdc2020_corsika.cfg:134]: It doesn't matter, the system just replace all the % variables, e.g. look at the dataset defintions:
```
job_output_2.add_to_dataset = sim.%(submitter)s.CosmicDSStopsCORSIKA.%(desc)s.art
```

Comment (brownd1978) [reply CampaignConfig/mdc2020_corsika.cfg:134]: OK, thanks for the explanation.

**Review by brownd1978 — APPROVED** [review]

**Pull request merged**